### ISSUE TITLE
Database error in combination with BuddyBoss Platform & BuddyBoss App plugin

### ISSUE THREAD

**Issue by mahedihasannoman** [buddyboss/buddyboss-platform#2628]
**Describe the bug**
Having a database error when BuddyBoss Platform & BuddyBoss App plugin are installed. See the error details below:

`ALTER TABLE wp_bb_performance_cache CHANGE COLUMN `id` id bigint(20) NOT NULL AUTO_INCREMENT PRIMARY KEY`

Caller:

```
BuddyBoss\P\Performance->on_activation()
wp-content/plugins/buddyboss-platform/bp-performance/classes/class-performance.php:261
BuddyBossApp\C\P\Performance->components_loaded()
wp-content/plugins/buddyboss-app/include/Components/Performance/Performance.php:58
do_action('bbapp_components_loaded')
wp-includes/plugin.php:484
bbapp->load_components()
wp-content/plugins/buddyboss-app/buddyboss-app.php:383
bbapp->load()
wp-content/plugins/buddyboss-app/buddyboss-app.php:181
do_action('plugins_loaded')
wp-includes/plugin.php:484
```
Also please see this video to replicate this issue: https://www.loom.com/share/543b63b3979f436eac8b86d77f262538

**To Reproduce**
1. Please see the above video.

**Environment**
local & Client's server

**Support ticket links**
https://secure.helpscout.net/conversation/1572619760/153599?folderId=4687016

**Comment by ExoGeek** [comment]
Hello,

Based on [https://codex.wordpress.org/Creating_Tables_with_Plugins](https://codex.wordpress.org/Creating_Tables_with_Plugins), you can't use `id bigint(20) NOT NULL AUTO_INCREMENT PRIMARY KEY` with dbdelta function.

This code : 

```
$sql[] = "CREATE TABLE {$bp_prefix}bp_notifications_meta (
		id bigint(20) NOT NULL AUTO_INCREMENT PRIMARY KEY,
		notification_id bigint(20) NOT NULL,
		meta_key varchar(255) DEFAULT NULL,
		meta_value longtext DEFAULT NULL,
		KEY notification_id (notification_id),
		KEY meta_key (meta_key(191))
	) {$charset_collate};";
```

Should be replaced by : 

```
$sql[] = "CREATE TABLE {$bp_prefix}bp_notifications_meta (
		id bigint(20) NOT NULL AUTO_INCREMENT,
		notification_id bigint(20) NOT NULL,
		meta_key varchar(255) DEFAULT NULL,
		meta_value longtext DEFAULT NULL,
		PRIMARY KEY  (`id`),
		KEY notification_id (notification_id),
		KEY meta_key (meta_key(191))
	) {$charset_collate};";
```

**Comment by marienel21** [comment]
Hi @ExoGeek 

The issue mentioned above is released in BuddyBoss Platform v1.7.4

You can check release notes here:
https://www.buddyboss.com/resources/buddyboss-platform-releases/1-7-4/

Please update the plugin to see the latest changes.

Thank you.
